.cookiecutter.json

```
@@ -4,20 +4,20 @@
         "full_name": "Network to Code, LLC",
         "email": "info@networktocode.com",
         "github_org": "nautobot",
-        "plugin_name": "nautobot_plugin_nornir",
-        "verbose_name": "Nautobot Nornir Plugin",
-        "plugin_slug": "nautobot-plugin-nornir",
-        "project_slug": "nautobot-plugin-nornir",
-        "repo_url": "https://github.com/nautobot/nautobot-plugin-nornir",
-        "base_url": "plugin-nornir",
+        "app_name": "nautobot_app_nornir",
```

Revert due to packaging issues with this app being non-conforming 
```suggestion
        "app_name": "nautobot_plugin_nornir",
```